src/main/java/trackitnus/model/util/SampleDataUtil.java

```diff
@@ -38,6 +39,8 @@
             new Contact(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
                 getTagSet("classmates", "CS2103T")),
             new Contact(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
+                getTagSet("colleagues")),
+            new Contact(new Name("Henry CHIA Wai Kit"), new Phone("65161556"), new Email("hchia@comp.nus.edu.sg"),
```

Comment:
Is the CHIA intended to be all caps? Haha

Reply:
Yeah I copied his name from SoC's website 😄